src/__test__/EditClass.test.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import Editclass from '../components/EditClass'
+import fetch from 'jest-fetch-mock'
+import { FAKE_SERVER_DATA } from '../test-data'
+import { shallow } from 'enzyme'
+
+global.fetch = fetch
+
+fetch.mockResponse(JSON.stringify(FAKE_SERVER_DATA))
+
+/* global it describe expect beforeEach jest */
+
+describe('Editclass', () => {
+  let wrapper, updateClassCallback, updatePrestigeCallback, app
+
+  beforeEach(() => {
+    updateClassCallback = jest.fn()
+    updatePrestigeCallback = jest.fn()
+
+    wrapper = shallow(<Editclass
+      activeCharacterClass='Fighter'
+      activeCharacterPrestige='Champion'
+      updateClass={updateClassCallback}
+      updatePrestige={updatePrestigeCallback} />)
+
+    wrapper.setState({klasses: {
```

Comment:
wow, this is a lot of content. Can't this be loaded from a file somewhere? Seems like it might be better off to have this test fixture as a loadable `.json` file.